feature/experimental/bgp/ate_tests/bgp_2byte_4byte_asn/bgp_2byte_4byte_asn_test.go

```diff
@@ -224,6 +227,7 @@ func createBgpNeighbor(nbr *bgpNbr, dut *ondatra.DUTDevice) *oc.NetworkInstance_
 	global := bgp.GetOrCreateGlobal()
 	global.As = ygot.Uint32(nbr.globalAS)
 	global.RouterId = ygot.String(dutSrc.IPv4)
+	global.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST).Enabled = ygot.Bool(true)
```

Comment:
It's would be good to provide a few example that `GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST).Enabled = ygot.Bool(true)` is a standardize way, and has been already used by other BGP tests. 

This will help the review make quick decision. Thanks!

Reply:
Sounds good, will do this moving forward, I'll provide examples such as other tests have implemented moving forward for quicker review process. i.e: https://github.com/openconfig/featureprofiles/blob/08cae3d2f4750d98ff210e7ef8323e8e71c53fda/feature/bgp/prefixlimit/otg_tests/bgp_prefix_limit_test/bgp_prefix_limit_test.go#L296-L297